CMakeLists.txt

```diff
@@ -1053,184 +1137,183 @@ set_property(TARGET irods_plugin_dependencies PROPERTY SOVERSION ${IRODS_VERSION
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcGeneralUpdate.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcSubStructFileGet.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcSubStructFilePut.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcSyncMountedColl.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcStructFileSync.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcOpenCollection.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcReadCollection.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcCloseCollection.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcCollRepl.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcRmColl.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcStructFileExtract.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcStructFileExtAndReg.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcStructFileBundle.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcChkObjPermAndStat.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcUserAdmin.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcGetRemoteZoneResc.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcDataObjOpenAndStat.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcL3FileGetSingleBuf.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcL3FilePutSingleBuf.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcDataObjCreateAndStat.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcFileStageToCache.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcFileSyncToArch.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcGeneralRowInsert.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcGeneralRowPurge.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcPhyBundleColl.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcUnbunAndRegPhyBunfile.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcGetHostForPut.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcGetRescQuota.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcBulkDataObjReg.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcBulkDataObjPut.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcEndTransaction.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcProcStat.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcStreamRead.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcSpecificQuery.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcStreamClose.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcGetHostForGet.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcDataObjLock.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcTicketAdmin.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcGetTempPasswordForOther.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcPamAuthRequest.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcSslStart.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcSslEnd.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcDataObjTruncate.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcFileTruncate.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcSubStructFileTruncate.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcAuthPluginRequest.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcGetLimitedPassword.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcGetHierarchyForResc.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcServerReport.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcZoneReport.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcClientHints.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcGetHierFromLeafId.cpp
-  ${CMAKE_SOURCE_DIR}/lib/api/src/rcIESClientHints.cpp
-  )
+#set(
```

Comment:
A comment explaining this section may be appropriate. Otherwise, should we just delete these commented sections?

Reply:
The commented lines are going to be removed. They are just left overs from experimentation.

Reply:
removed commented lines.